app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductBrowsingListActivity.java

```diff
@@ -75,7 +75,7 @@ protected void onCreate(Bundle savedInstanceState) {
         Bundle extras = getIntent().getExtras();
 
         typeStrings = new String[]{
-                "brand", "country", "additive", "search","store","packaging"
+                "brand", "country", "additive", "search", "store", "packaging", "label"
         };
```

Comment:
Hey @PrajwalM2212 can you add categories as well in this PR and solve similar issues in the same PR.